Editor/Data/Attribute/RestrictedAttributeDrawer.cs

```diff
@@ -0,0 +1,63 @@
+﻿namespace Zinnia.Data.Attribute
+{
+    using UnityEngine;
+    using UnityEditor;
+
+    [CustomPropertyDrawer(typeof(RestrictedAttribute))]
+    public class RestrictedAttributeDrawer : PropertyDrawer
+    {
+        public override void OnGUI(Rect position, SerializedProperty property, GUIContent label)
+        {
+            EditorGUI.BeginProperty(position, label, property);
+            RestrictedAttribute attrib = attribute as RestrictedAttribute;
+
+            bool playingAndActiveAndEnabled = Application.isPlaying && property.serializedObject.targetObject is Behaviour behaviour && behaviour.isActiveAndEnabled;
```

Comment:
```suggestion
            bool isPlayingAndActiveAndEnabled = Application.isPlaying && property.serializedObject.targetObject is Behaviour behaviour && behaviour.isActiveAndEnabled;
```
Regular C# naming convention for boolean types.